data/regions.js

```diff
@@ -0,0 +1,36 @@
+const raw = require('./regions.json');
+
+const roots = [];
+const lookup = raw;
+
+Object.values(raw).forEach((region) => {
+  // eslint-disable-next-line no-param-reassign
+  region.children = region.children || [];
+
+  const last = region.regionKey.lastIndexOf('-');
+  if (last < 0) {
+    // eslint-disable-next-line no-param-reassign
+    region.parentRegionKey = null;
+    roots.push(region.regionKey);
+    return;
+  }
+
+  const parentRegionKey = region.regionKey.substr(0, last);
+
+  // eslint-disable-next-line no-param-reassign
```

Comment:
Something cool, maybe you already know, but you can also do `// eslint-disable-file no-param-reassign` once at the top of the file instead of repeating it.

Reply:
I wanted to be more selective to avoid it as much as I could.